cli/src/plugins/javadoc/buildIndexes.ts

```diff
@@ -39,4 +39,25 @@ export const buildIndexes = async ({
     path: "/index",
     root,
   });
+
+  const packages = finalizedProject.entities
+    .filter((e) => e.data?.category === "package")
+
+  packages.forEach(pkg => {
+    const pkgRoot = md.root([
+      toctree(finalizedProject.entities.filter((e) => e.data?.category === "class").filter((e) => e.data?.containingPackage === pkg.canonicalName)
+        .map(child => {
+          const sanitizedName = child.canonicalName.replace(/#.*$/, "");
+          const value = sanitizedName.substring(sanitizedName.lastIndexOf('.') + 1) // only want simple class name
+          return toctreeItem({
+            url: child.canonicalName.replace(/\./g,'/'),
```

Comment:
This is rewritten a lot. It would be good to centralize in a standalone function so that we don't end up with a typo in one place that causes massive headaches as we try to figure out why the links aren't lining up.

Reply:
Agreed. I wasn't sure where to put it before, but it lives in `buildIndexes.ts` for now.